src/directives/test-directive/test.directive.js

```diff
@@ -1,15 +1,17 @@
+import template from './test.directive.html';
+import './test.directive.scss';
+
 export default ngModule => {
   if (TEST_MODE) {
     require('./test.directive.test').default(ngModule);
   }
-  require('./test.directive.scss');
 
   ngModule.directive('testDirective', $log => {
     return {
       restrict: 'E',
       scope: {},
       // templateUrl: 'src/directives/test.directive.html',
-      template: require('./test.directive.html'),
+      template,
```

Comment:
See this [issue](https://stackoverflow.com/questions/57482205/loading-angularjs-html-templates-in-webpack-4-renders-object-module) for why this was changed. We could instead:
```javascript
require('./test.directive.html').default
```